core/src/main/java/org/zenoss/zep/dao/impl/EventTimeDaoImpl.java

```diff
@@ -99,7 +100,7 @@ public List<Zep.EventTime> findProcessedSince(Date startDate, int limit) {
         String sql = "SELECT * from %s where %s >= :since order by %s asc limit %s";
         sql = String.format(sql, TABLE_EVENT_TIME, COLUMN_PROCESSED, COLUMN_PROCESSED, limit);
 
-        return template.query(sql, new EventTimeRowMapper(), params);
+        return template.query(sql, params, new EventTimeRowMapper());
```

Comment:
can we name this 'fields' instead of 'params' for consistency with everywhere else?  Or the other way around if you like...